service-template/.github/workflows/{{project-name}}.yml

```diff
@@ -0,0 +1,158 @@
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - '{{project-name}}/v*'
+  pull_request:
+    paths:
+      - {{project-name}}/**
+      - .github/workflows/{{project-name}}.yml
+      - .github/actions/deps/**
+
+name: Test, lint and docker ({{project-name}})
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: blockscout/{{project-name}}
+
+defaults:
+  run:
+    working-directory: {{project-name}}
+
+jobs:
+  test:
+    name: Unit, doc and integration tests
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: admin
+          POSTGRES_USER: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
```

Comment:
Can we setup the database initialization depending on whether the service needs to connect to postgresql database or not?